webapp/app/compareperf.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html ng-app="compare">
+<head>
+    <meta charset="iso-8859-1" />
+    <title>Compare Talos</title>
```

Comment:
Let's call this `Compare Performance` as we'll be using this for things aside from Talos at some point likely in the not too distant future.